uint8 cmd
float32 val1
float32 val2
float32 duration
---
bool state
